### DISCUSSION TITLE
No docker images available for latest releases

### DISCUSSION THREAD

**Discussion by esbakker** [apiman/apiman#2168] [Help] [answered]
We use the [wildfly-manager](https://hub.docker.com/r/apiman/wildfly-manager) and [vertx-openshift](https://hub.docker.com/r/apiman/vertx-openshift) docker containers. There is a vertx-openshift container for the latest (2.2.3) version, but the latest wildfly-manager container still has version 2.1.2. We need 2.2.3 for the fix of https://github.com/apiman/apiman/issues/1829.

(Also the vertx-openshift has tags for different versions, while wildfly-manager only has a latest tag. It would be nice if that one would have version tags too).

**Comment by msavy** [comment, marked as answer]
Hi, finally got some free time to work on this. 

I hadn't realised that this was not being updated properly (various updates were done to release scripts it seems).

Can you please check whether this is working fine for you now?

**Reply by msavy** [reply]
Should be available now

**Comment by esbakker** [comment]
Thanks, works like a charm :).

**Reply by msavy** [reply]
Thanks, Eddy!
